addons/website_blog/views/website_blog_posts_loop.xml

```diff
@@ -136,8 +136,9 @@ according to the enabled options.
                                 <t t-call="website_blog.post_info"></t>
                             </div>
                         </t>
-                        <!-- Add 'unpublished' badge -->
-                        <span t-if="not blog_post.website_published" class="bg-danger small py-1 px-2 position-absolute o_not_editable" style="top:0; right:0">unpublished</span>
+                        <!-- Add 'unpublished' or 'scheduled' badge -->
+                        <span t-if="blog_post.scheduled" class="bg-warning small py-1 px-2 position-absolute o_not_editable" style="top:0; right:0">Scheduled</span>
+                        <span t-elif="not blog_post.website_published" class="bg-danger small py-1 px-2 position-absolute o_not_editable" style="top:0; right:0">Unpublished</span>
```

Comment:
Same as in the other file, you could use a `t-set` for the only things that change (the class and the word) rather than copying the whole element.